src/style/loqui/index.css

```diff
@@ -1651,6 +1655,18 @@ article.form ul.listBox li.add:before {
     border: none;
     border-radius: 0;
   }
+  section#chat #footbox #typing {
+    position: fixed;
+    z-index: 9;
+    bottom: 0.1rem;
+    top: auto;
+  }
+  button {
+    z-index: 9;
+  }
+  section#chat nav#plus {
+    display: none;
+  }
```

Comment:
+ #typing: the typing notification would otherwise be hidden
+ button: otherwise not clickable for some reason, although visible in the UI
+ hide the plus bar in corner cases where the code solution is not applied